packages/dds/matrix/test/matrix.spec.ts

```diff
@@ -6,7 +6,7 @@
 import "mocha";
```

Comment:
A little opportunistic clean up in the unit tests.  These tests are still the same.